packages/starlight/components/Icons.ts

```diff
@@ -123,6 +123,7 @@ export const BuiltInIcons = {
 		'<path d="M12 10.8c-1-2.1-4-6-6.8-8C2.6 1 1.6 1.3.9 1.6.1 1.9 0 3 0 3.8c0 .7.4 5.6.6 6.4C1.4 13 4.3 14 7 13.6h.4H7c-4 .6-7.4 2-2.8 7 5 5.3 6.8-1 7.8-4.2 1 3.2 2 9.3 7.7 4.3 4.3-4.3 1.2-6.5-2.7-7a9 9 0 0 1-.4-.1h.4c2.7.3 5.6-.6 6.4-3.4.2-.8.6-5.7.6-6.4 0-.7-.1-1.9-.9-2.2-.7-.3-1.7-.7-4.3 1.2-2.8 2-5.7 5.9-6.8 8"/>',
 	discourse:
 		'<path d="M12.102 0h-.081C5.462 0 .13 5.252.001 11.779v.012L.007 24l12.097-.01c6.582-.055 11.897-5.404 11.897-11.995S18.686.056 12.109 0h-.005zM12 18.857h-.015a6.778 6.778 0 0 1-2.94-.666l.041.018-4.345 1.077 1.227-4.018a6.78 6.78 0 0 1-.83-3.27A6.86 6.86 0 1 1 12 18.857z"/>',
+	zulip: '<path d="M20.972,19.009c-0,1.645 -1.207,2.991 -2.682,2.991l-12.58,-0c-1.475,-0 -2.682,-1.346 -2.682,-2.991c0,-1.008 0.453,-1.903 1.142,-2.446l6.694,-5.977c0.125,-0.107 0.286,0.065 0.196,0.208l-2.455,4.916c-0.069,0.138 0.02,0.307 0.162,0.307l9.523,0c1.475,0 2.682,1.347 2.682,2.992Zm-0,-14.018c-0,1.008 -0.453,1.903 -1.142,2.446l-6.694,5.977c-0.125,0.107 -0.286,-0.065 -0.196,-0.208l2.455,-4.916c0.069,-0.138 -0.02,-0.307 -0.162,-0.307l-9.523,-0c-1.475,-0 -2.682,-1.346 -2.682,-2.992c0,-1.645 1.207,-2.991 2.682,-2.991l12.58,0c1.475,0 2.682,1.346 2.682,2.991Z"/>',
```

Comment:
Ran this through [SVGOMG](https://jakearchibald.github.io/svgomg/) and was able to shave a few more bytes off:

```suggestion
	zulip: '<path d="M21 19c0 1.7-1.2 3-2.7 3H5.7C4.2 22 3 20.7 3 19a3 3 0 0 1 1.2-2.4l6.7-6c0-.1.3 0 .2.2l-2.5 5s0 .2.2.2h9.5c1.5 0 2.7 1.4 2.7 3Zm0-14a3 3 0 0 1-1.2 2.4l-6.7 6c0 .1-.2 0-.2-.2l2.5-5s0-.2-.2-.2H5.7C4.2 8 3 6.6 3 5c0-1.7 1.2-3 2.7-3h12.6C19.8 2 21 3.3 21 5Z"/>',
```

Reply:
Ahh, awesome!

Reply:
For good measure I put both icons side-by-side and I can't see a difference, even at really high zoom levels

![Two Zulip icons side-by-side (they look indistinguishable)](https://github.com/withastro/starlight/assets/1362179/6e928fe7-8835-4a93-b600-1019d7ee28d1)